entrypoint.sh

```diff
@@ -21,9 +21,11 @@ set -e
 
 echo "First Install";
 ls -alh
-echo "Install Yarn"
-[ -f yarn.lock ] && yarn install --frozen-lockfile --prefer-offline
-[ -f package-lock.json ] && npm install
+if [ "$2" = true ] ; then
+    echo "Install Yarn"
+    [ -f yarn.lock ] && yarn install --frozen-lockfile --prefer-offline
+    [ -f package-lock.json ] && npm install
```

Comment:
Then use the `npm ci` command which is basically similar to what i did with the `yarn`, just not as familiar with `npm` anymore.

```suggestion
    [ -f package-lock.json ] && npm ci
```

Reply:
I'm not fully aware of the difference between `npm ci` either. I think `npm ci` doesn't use the latest package available, but the package version of the current `package-lock.json`. I guess is a bit safer.

I leave this decision to you, but I think `npm install` is not a bad choice either 👍

Reply:
`npm ci` is synonymous with `yarn install --frozen-lockfile` so prob the way to go